src/aioquic/quic/recovery.py

```diff
@@ -62,7 +63,10 @@ def update_bucket(self, now: float) -> None:
             self.evaluation_time = now
 
     def update_rate(self, congestion_window: int, smoothed_rtt: float) -> None:
-        pacing_rate = congestion_window / max(smoothed_rtt, K_MICRO_SECOND)
+        if self.pacing_rate == None or self.pacing_rate == 0:
+            pacing_rate = congestion_window / max(smoothed_rtt, K_MICRO_SECOND)
+        else:
+            pacing_rate = self.pacing_rate
```

Comment:
Why are we changing this?

Reply:
Woopsie, seems like an artifact of BBR (where the congestion controller needed to set the pacing rate), just removed it